winml/adapter/winml_adapter_dml.cpp

```diff
@@ -22,30 +22,25 @@ namespace winmla = Windows::AI::MachineLearning::Adapter;
 
 EXTERN_C IMAGE_DOS_HEADER __ImageBase;
 
-static std::string CurrentModulePath() {
-  char path[MAX_PATH];
-  FAIL_FAST_IF(0 == GetModuleFileNameA((HINSTANCE)&__ImageBase, path, _countof(path)));
```

Comment:
why no more fail fasts?

Reply:
oh hm, because i had validated on my machine which didnt have the FF macros, adding them back in.

Reply:
Fixed